src/CMakeLists.txt

```diff
@@ -1,14 +1,7 @@
 # See URLs below for more documentation on ExternalProject
 # http://www.cmake.org/cmake/help/v3.0/module/ExternalProject.html
 # http://www.kitware.com/media/html/BuildingExternalProjectsWithCMake2.8.html
-
-ExternalProject_Add(libfunctionality
-    SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/libfunctionality"
-    INSTALL_DIR "${ANDROID_INSTALL_DIR}"
-    CMAKE_GENERATOR "${ANDROID_GENERATOR}"
-    CMAKE_ARGS ${COMMON_CMAKE_ARGS} # put any others here
-    BUILD_IN_SOURCE 0)
-
```

Comment:
So order here doesn't really matter for non-dependent packages - you have to manually specify dependencies to control actual build order.